src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/FSharp.BrowseObject.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright, Microsoft Corporation, All rights reserved.-->
```

Comment:
Should we just unify these between C#/VB and F#? Do they have any different properties?